src/javascript/typescript/__tests__/codeGeneration.ts

```diff
@@ -22,7 +22,7 @@ function compile(
   return compileToIR(schema, document, options);
 }
 
-describe('Typescript codeGeneration', () => {
+describe.only('Typescript codeGeneration', () => {
```

Comment:
i don't think you want this describe.only here

Reply:
whoops!